src/layout/TextHighlight.js

```diff
@@ -3,11 +3,19 @@ import * as F from 'futil-js'
 import { withTheme } from '../utils/theme'
 
 // Since start and end are the same token, splitting on it means every even element was a match
-let TextHighlight = ({ pattern, text, theme: { HighlightWrap } }) =>
+let TextHighlight = ({ pattern, text, theme: { HighlightWrap }, style }) =>
```

Comment:
Why add style support here? That's what overriding HighlightWrap is for

Reply:
Just a note, I believe this was because the only difference between the GV TextHighlight and the base layout one was the style prop. Doesn't matter now anyway ¯\\\_(ツ)\_/¯